ReactCommon/turbomodule/React-turbomodule.podspec

```diff
@@ -0,0 +1,49 @@
+# coding: utf-8
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "..", "..", "package.json")))
+version = package['version']
+
+source = { :git => 'https://github.com/facebook/react-native.git' }
+if version == '1000.0.0'
+  # This is an unpublished version, use the latest commit hash of the react-native repo, which we’re presumably in.
+  source[:commit] = `git rev-parse HEAD`.strip
+else
+  source[:tag] = "v#{version}"
+end
+
+folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
+folly_version = '2018.10.22.00'
+boost_compiler_flags = '-Wno-documentation'
+
+Pod::Spec.new do |s|
+  s.name                   = "React-turbomodule"
+  s.version                = version
+  s.summary                = "-"  # TODO
+  s.homepage               = "http://facebook.github.io/react-native/"
+  s.license                = package["license"]
+  s.author                 = "Facebook, Inc. and its affiliates"
+  s.platforms              = { :ios => "9.0", :tvos => "9.2" }
+  s.source                 = source
+  s.source_files           = "**/*.{cpp,h}"
+  s.exclude_files          = "**/android/*"
+  s.compiler_flags         = folly_compiler_flags + ' ' + boost_compiler_flags
+  s.pod_target_xcconfig    = { "HEADER_SEARCH_PATHS" => "\"$(PODS_ROOT)/boost-for-react-native\" \"$(PODS_ROOT)/Folly\"" }
+  s.header_dir             = "jsireact"
+
+  s.dependency "React-Core", version
+  s.dependency "React-cxxreact", version
+  s.dependency "React-jsi", version
+  s.dependency "Folly", folly_version
+
+  s.subspec "RCTTurboModule" do |ss|
+    ss.source_files   = "**/*.{mm,h}"
```

Comment:
should probably include cpp here too (even if it doesn't exist today)

Reply:
Also, would it be possible to have 2 parts of this definition?
1. the shared part, under the turbomodule/core/*
2. the platform-specific part, under turbomodule/core/platform/ios/*

Sample:
1. https://github.com/facebook/react-native/blob/e6f7d69428a4b3344ee5205c11bca58cbb4317fc/ReactCommon/turbomodule/core/BUCK#L6-L13
2. https://github.com/facebook/react-native/blob/e6f7d69428a4b3344ee5205c11bca58cbb4317fc/ReactCommon/turbomodule/core/BUCK#L47-L61

You should probably move this podspec file to turbomodule/core/ subdir to mirror BUCK location

Reply:
The shared part is the top level podspec, RCTTurboModule is the platform specific part. I can lock down the dirs being searched so it is clear.

Reply:
I see, let's then call the subspec `core` instead of `RCTTurboModule`.